accname/index.html

```diff
@@ -671,6 +671,13 @@ <h4>Computation steps</h4>
                       <span id="step2F.iii"><!-- Don't link to this legacy numbered ID. --></span><em>Name From Each Child:</em> For each <code>rendered child node</code> of the
                       <code>current node</code>:
                       <ol>
+                        <li id="comp_name_from_content_find_child_actions">
```

Comment:
Do we need to say both of these things? They seem like they're essentially saying the same as each other.